packages/driver/src/cypress/mocha.ts

```diff
@@ -73,10 +117,17 @@ export function calculateTestStatus (test: CypressTest, strategy: Strategy, opti
     failedTests.push(test)
   }
 
+  if (!test.prevAttempts?.length) {
+    test.thisAttemptInitialStrategy = 'NONE'
```

Comment:
`thisAttemptInitialStrategy` is never referenced, from what I can tell - is this a field we need to be reporting to the cloud?

Reply:
Yes, see the description [here](https://cypress-io.atlassian.net/browse/CYCLOUD-1665)